src/controllers/graphql/user-controller.js

```diff
@@ -14,6 +14,10 @@ export const deleteUser = (_, id, { models }) => {
   })
 }
 
+export const currentUser = (_, id, { user }) => {
+  return user
+}
```

Comment:
You can write this in one line with shorter syntax:
```suggestion
export const currentUser = (_, params, { user }) => user
```